client/my-sites/plugins/plugin-item/plugin-item.jsx

```diff
@@ -132,14 +133,21 @@ module.exports = React.createClass( {
 			showDismiss={ false } />;
 	},
 
-	renderActions: function() {
+	renderActionsOrCount: function() {
```

Comment:
I think this could look nicer if we write two separate methods `renderActions` and `renderCount` and we put the conditional on the render method, eg. `{ selectedSite && this.renderActions }` :)